gazebo/rendering/LensFlare.cc

```diff
@@ -279,19 +283,41 @@ namespace gazebo
           pos.y /= pos.w;
           // check if light is visible
           if (std::fabs(pos.x) <= 1 &&
-              std::fabs(pos.y) <= 1 && pos.z > abs(pos.w))
+              std::fabs(pos.y) <= 1 && pos.z > -abs(pos.w))
           {
+            // The ogreEnvCamera and wideAngleDummyCamera used here both
+            // transform from gazebo z-up coords to Ogre y-up coords. If the
```

Comment:
In retrospect, describing this as "transform from gazebo z-up coords to Ogre y-up coords" might not be accurate. Any insight into what this transform is really doing would be appreciated. It can be found here https://github.com/gazebosim/gazebo-classic/blob/gazebo11/gazebo/rendering/Camera.cc#L1585-L1586 and here https://github.com/gazebosim/gazebo-classic/blob/gazebo11/gazebo/rendering/WideAngleCamera.cc#L590-L591